### PR TITLE
Em 97

### PR DESCRIPTION
**Pull request by lalodiazdiaz**

### CODE REVIEW

File: src/components/Dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
 
 function Dashboard() {
 	const loggedUser = window.localStorage.getItem('user');
 	const userLogged = JSON.parse(loggedUser);
 	const [state, setState] = useState(false);
 	const NAME = userLogged.data.fullName;
+	const dispatch = useDispatch();
+	const local = JSON.parse(localStorage.getItem('user'));
+	const { token } = local.data;
+	const AuthStr = `Bearer ${token}`;
```
Comment (FraGa96) [inline src/components/Dashboard/Dashboard.jsx]: why is this here? would be better in the service that makes the backen call

**Review by FraGa96 — CHANGES_REQUESTED** [review]

**Review by FraGa96 — APPROVED** [review]

**Review by LuzLimon — APPROVED** [review]
